web/src/components/core/InstallationFinished.test.jsx

```diff
@@ -60,4 +72,47 @@ describe("InstallationFinished", () => {
     await user.click(rebootButton);
     expect(finishInstallationFn).toHaveBeenCalled();
   });
+
+  describe("when TPM is set as encryption method", () => {
+    beforeEach(() => {
+      encryptionMethod = EncryptionMethods.TPM;
+    });
+
+    describe("and encryption was set", () => {
+      it("shows the TPM reminder", async () => {
+        installerRender(<InstallationFinished />);
+        await screen.findAllByText(/TPM/);
+      });
+    });
+
+    describe("but encryption was not set", () => {
+      beforeEach(() => {
+        encryptionPassword = "";
+      });
+
+      it("does not show the TPM reminder", async () => {
+        const { user } = installerRender(<InstallationFinished />);
+        // Forcing the test to slow down a bit with a fake user interaction
+        // because actually the reminder will be not rendered immediately
+        // making the queryAllByText to produce a false positive if triggered
+        // too early here.
+        const congratsText = screen.getByText("Congratulations!");
+        await user.click(congratsText);
+        expect(screen.queryAllByText(/TPM/)).toHaveLength(0);
```

Comment:
BTW, today I _learned_ that another option for doing this might be to add a helper function in test-uitls. Something like `slowDown`, `smallWait` or whatever other name suits better

```js

const shortBreak = async (ms = 500) => new Promise(resolve => setTimeout(resolve, ms));
```


and then in the test

```js
import { shortBreak } from "test-utils";

...

await shortBreak();
```

I haven't tested it, just got _inspiration_ from [react-spectrum](https://github.com/adobe/react-spectrum/blob/55aca809c4ce039efe672aaced8c77553d5c03f5/packages/%40react-spectrum/list/stories/ListView.stories.tsx#L327-L328)